inception_reports/generate_reports_manager.py

```diff
@@ -320,67 +657,105 @@ def login_to_inception(api_url, username, password):
     return False, None
 
 
-
-def set_sidebar_state(value):
-    if(st.session_state.sidebar_state == value):
-        st.session_state.flag = value
-        st.session_state.sidebar_state = "expanded" if value == "collapsed" else "collapsed"
-    else:
-        st.session_state.sidebar_state = value
-    st.rerun()
-
-
-
 def select_method_to_import_data():
     """
     Allows the user to select a method to import data for generating reports.
     """
-    
-    method = st.sidebar.radio("Choose your method to import data:", ('Manually', 'API'), index=0)
 
-    if method == 'Manually':
-        st.sidebar.write("Please input the path to the folder containing the INCEpTION projects.")
-        projects_folder = st.sidebar.text_input("Projects Folder:", value="data/dresden_projects/")
+    method = st.sidebar.radio(
+        "Choose your method to import data:", ("Manually", "API"), index=0
+    )
+
+    if method == "Manually":
+        st.sidebar.write(
+            "Please input the path to the folder containing the INCEpTION projects."
+        )
+        projects_folder = st.sidebar.text_input(
+            "Projects Folder:", value="data/dresden_projects/"
```

Comment:
Please void mentioning names or people, location, etc. in the code. Maybe no value should be hard-coded here at all?